packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -72,7 +72,11 @@ export const _videoGalleryRemoteParticipantsMemo: _VideoGalleryRemoteParticipant
             participant.raisedHand,
             participant.contentSharingStream,
             remoteParticipantReaction,
-            spotlight
+            spotlight,
+            Math.max(
+              (participant.diagnostics?.networkReceiveQuality?.value ?? 0) as number,
```

Comment:
why is `as number` needed?

Reply:
Looks like this could also be `boolean`??
![image](https://github.com/user-attachments/assets/f6b599ae-a887-42c8-a69d-5dc1606653eb)

Reply:
For using DiagnosticQuality vs number - it is for getting component using as less sdk types as possible